api/services/program_sequence_service.py

```diff
@@ -0,0 +1,62 @@
+from api import models, helpers
+
+
+class ProgramSequenceService(object):
+    def __init__(self):
+        self.content_id = None
+        self.user_phone = None
+
+    def set_init_data(self, jsonData):
+        self.content_id = jsonData["content_id"]
+        user_phone = helpers.fetch_by_key("urn", jsonData["contact"])
+        self.user_phone = helpers.sanitize_phone_string(user_phone)
+
+    def get_program_sequence_id(self, jsonData):
+        self.set_init_data(jsonData)
+        try:
+            user_details = models.User.query.get_by_phone(self.user_phone)
+            if not user_details:
+                # Need to log this
+                print("User not available")
+                return None, None
+
+            module_content_details = models.ModuleContent.query.get_by_content_id(
+                self.content_id
+            )
+
+            if not module_content_details:
+                # it is optional to have Content and Module association
+                return None, None
```

Comment:
Why returning 2 parameter and you are expecting only one.

Reply:
Updated